creusot-contracts/src/ghost/resource.rs

```diff
@@ -1,5 +1,16 @@
 //! Resource algebras
-pub mod fmap_view;
+//!
+//! See [`Resource`].
+//!
+//! This module also defines a specialization of [`Resource`] for the case where you want:
+//! - A single, authoritative version of a [`FMap`][crate::logic::FMap]
+//! - Multiple fragments of such a map, that each assert that some key-value is in the
+//!   map.
+//!
+//! These are the [`Authority`] and [`Fragment`] types respectively.
+mod fmap_view;
+
+pub use fmap_view::{Authority, Fragment};
```

Comment:
I'm not so sure for this change. `Authority` and `Fragment` really come together, and comes at a different level from `Ressource`.